SU-SWS/ace-casbsgryphon
# CAS-180: updated our view, bruce's view, added a qa view so I can see…

… his data

# READY FOR REVIEW

# Summary
-  Changes to Stanford person view so we can filter by casbs year
- Changes to CASBS Current Fellows view so that it is extendable
- Addition of a QA view so I can see what the data is like

# Review By (Date)
- ASAP

# Urgency
- I need it to unblock work (esp. that QA view)

# Steps to Test

1.  Check out those views
2. Maybe also go look at /admin/casbs-people-qa so you can see what I am QAing

# Affected Projects or Products
- All the people pages

# Associated Issues and/or People
- CAS-180
- CAS-181

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
